.github/workflows/docs-ci.yml

```diff
@@ -33,5 +33,16 @@ jobs:
       - name: Check for documentation style errors
         working-directory: ./docs
         run: ./scripts/doc8_style_check.sh
+        
+      - name: Set up Go
```

Comment:
You cannot run the tests as part of the documentation workflow. Either create a new workflow or use the azure pipeline instead.